src/amo/components/GuidesAddonCard/styles.scss

```diff
@@ -21,6 +21,11 @@
     font-size: $font-size-m;
     font-weight: 500;
     margin-bottom: 0;
+
+    a {
+      color: $black;
+      text-decoration: none;
```

Comment:
Do you think we should make the link change to blue on hover, like we do, for example, with search results?

Reply:
oh? let me check..

Reply:
yes, I like that 👍  thanks for the suggestion